### PR TITLE
IE9 bugfix for loading SVGs from URL that contain <text> nodes

### PR DESCRIPTION
**Pull request by mbrzuzy**

### CODE REVIEW

File: src/shapes/text.class.js [hunk 1]
```diff
@@ -1038,8 +1038,25 @@
...
+      if ('firstChild' in element && element.firstChild !== null) {
+        if ('data' in element.firstChild && element.firstChild.data !== null) {
+          textContent = element.firstChild.data;
+        }
+      }
+    } else {
+      textContent = element.textContent;
+    }
+
+    textContent = element.textContent.replace(/^\s+|\s+$|\n+/g, '').replace(/\s+/g, ' ');
```
Comment (asturur) [inline src/shapes/text.class.js]: you are accessing element.textContent here anyway. So the problem is still here.

**Comment by asturur** [comment]
Also textContent, altough not standard, takes all the content of the element/ node.
I think that firstChild.data takes the text of the first textNode of the element, discarding near tspans.
am i wrong?

File: src/shapes/text.class.js (hunk 1 above)
Comment (mbrzuzy) [inline src/shapes/text.class.js]: My mistake, I missed that when porting changes over.

**Comment by mbrzuzy** [comment]
Yes, you're right asturur. Although this fix no longer breaks fabric completely in IE9, tspans should still be supported. When I get the chance I'll amend my commit.

File: src/shapes/text.class.js (hunk 1 above)
Comment (kangax) [inline src/shapes/text.class.js]: Btw, we should probably use trim here — https://github.com/kangax/fabric.js/blob/f0fcd770bd489bb6c2322f99462c23a41b7ca6b9/src/util/lang_string.js#L10

**Comment by asturur** [comment]
here is missing somethig:
We have to concatenate all childs not jsut firstChild.

**Pull request merged**